RELEASE/scripts/autoscend/combat/auto_combat_ed.ash

```diff
@@ -118,58 +115,58 @@ string auto_edCombatHandler(int round, monster enemy, string text)
 		}
 	}
 
+	//iotm back item and the enemies it spawns (free fights) can be killed using special skills to get extra XP and item drops
 	if(have_equipped($item[Protonic Accelerator Pack]) && isGhost(enemy))
 	{
-		if(canUse($skill[Summon Love Gnats]))
+		skill stunner = getStunner(enemy);
+		if(stunner != $skill[none])
 		{
-			return useSkill($skill[Summon Love Gnats]);
+			return useSkill(stunner);
 		}
 
-		if(auto_have_skill($skill[Shoot Ghost]) && (my_mp() > mp_cost($skill[Shoot Ghost])) && !contains_text(edCombatState, "shootghost3") && !contains_text(edCombatState, "trapghost"))
```

Comment:
I'm not familiar with this IOTM and its code. TrappedGhost and having enough MP is no longer checked. Intentional?

Reply:
canUse includes a check for both having the skill and having enough MP to cast it. It also checks single use skills to see if they have been marked as "already used" in combatstate automatically.
canUse and useSkill thus replace 3 different manual checks, and do so more cleanly.

The only place to leave a MP check is if we are checking for "only use it if we have 3x more MP than needed to cast this skill" and the like. if it is only checking exact amount then it should use canUse instead